VideoApi/VideoApi.UnitTests/Controllers/Conference/GetConferenceDetailsByIdTests.cs

```diff
@@ -1,35 +1,45 @@
 using System;
+using System.Collections.Generic;
+using System.Linq;
 using System.Net;
 using FluentAssertions;
 using Microsoft.AspNetCore.Mvc;
 using Moq;
 using NUnit.Framework;
-using VideoApi.DAL.Queries;
+using VideoApi.Contract.Requests;
 using Task = System.Threading.Tasks.Task;
 
 namespace VideoApi.UnitTests.Controllers.Conference
 {
-    public class GetConferenceDetailsByIdTests : ConferenceControllerTestBase
+    public class GetConferencesTodayByHearingVenueNameTests : ConferenceControllerTestBase
     {
+        private readonly  string _hearingVenueName = "MyVenue";
+        private readonly ConferenceForAdminRequest _conferenceForAdminRequest = new ConferenceForAdminRequest
+        {
+            HearingVenueNames = new List<string>(),
+        };
         [Test]
         public async Task Should_return_ok_result_for_given_conference_id()
         {
-            var result = await Controller.GetConferenceDetailsByIdAsync(TestConference.Id);
+
+            var hearingVenueNames = new List<string>(new string[] { _hearingVenueName });
+
+            _conferenceForAdminRequest.HearingVenueNames = hearingVenueNames;
+            var result = await Controller.GetConferencesTodayForAdminByHearingVenueNameAsync(_conferenceForAdminRequest);
 
             var typedResult = (OkObjectResult)result;
             typedResult.StatusCode.Should().Be((int)HttpStatusCode.OK);
         }
 
         [Test]
-        public async Task Should_return_notfound_with_no_matching_conference()
+        public async Task Should_return_not_found_with_no_matching_conference()
         {
-            QueryHandlerMock
-                .Setup(x => x.Handle<GetConferenceByIdQuery, VideoApi.Domain.Conference>(It.IsAny<GetConferenceByIdQuery>()))
-                .ReturnsAsync((VideoApi.Domain.Conference)null);             
+            var hearingVenueNames = new List<string>(new string[] { It.IsAny<string>() });
 
+            _conferenceForAdminRequest.HearingVenueNames = hearingVenueNames;
+            var result = await Controller.GetConferencesTodayForAdminByHearingVenueNameAsync(_conferenceForAdminRequest);
 
-            var result = await Controller.GetConferenceDetailsByIdAsync(Guid.NewGuid());
-            var typedResult = (NotFoundResult)result;
+            var typedResult = (OkObjectResult)result;
```

Comment:
Should this not return a NotFoundResult or NotFoundObjectResult? And it might be worth asserting the status code is 404.

Reply:
If not then this test should be removed.